spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  before(:each) do
+    @or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
+    @ny_dmv_office_locations = DmvDataService.new.ny_dmv_office_locations
+    @mo_dmv_office_locations = DmvDataService.new.mo_dmv_office_locations
+    @oregon_facilities = FacilityFactory.new.create_facility_oregon(@or_dmv_office_locations)
+    @new_york_facilities = FacilityFactory.new.create_facility_new_york(@ny_dmv_office_locations)
+    @missouri_facilities = FacilityFactory.new.create_facility_missouri(@mo_dmv_office_locations)
+  end
+
+  describe '#initialize' do
+    it 'can initialize' do
+      factory = FacilityFactory.new
+      expect(factory).to be_an_instance_of(FacilityFactory)
+    end
+  end
+
+  describe '#create_facility_oregon' do
+    it 'can create facilities from Oregon DMV office data' do
+      expect(@oregon_facilities).to be_a(Array)
+      expect(@oregon_facilities.count).to eq(59)
```

Comment:
For now. This would break if the API we're pulling the data from added a facility. Makes our tests a bit brittle because they are dependent on data we don't control.